src/SFA.DAS.ApprenticeCommitments/DTOs/ApprenticeshipDtoMapping.cs

```diff
@@ -43,7 +46,42 @@ public static class ApprenticeshipDtoMapping
                 ApprovedOn = latest.CommitmentsApprovedOn,
                 ChangeOfCircumstanceNotifications = apprenticeship.ChangeOfCircumstanceNotifications,
                 StoppedReceivedOn = latest.StoppedReceivedOn,
+                Revisions = GetRevisions(apprenticeship.Revisions),
             };
         }
+
+        private static List<RevisionDto> GetRevisions(IReadOnlyCollection<Revision> revisions)
+        { 
+            var ret = new List<RevisionDto>();
+            if (revisions.Count == 1)
+                return ret;
+
+            revisions.ToList().OrderByDescending(x => x.Id).ToList().Pairwise((x, y) =>
+            {
+                if (x.Details.DeliveryModel != y.Details.DeliveryModel)
+                    ret.Add(new RevisionDto("Delivery model changed",                                                
+                         $"Delivery model changed from {y.Details.DeliveryModel} to {x.Details.DeliveryModel}",
+                         x.CreatedOn));
```

Comment:
If x the first and y, then shouldn't we be specifying the date created as the second value not the previous revision

Reply:
The list is OrderByDescending as the Jira states that entries should be displayed with the latest first.  so the x (latest) holds the date that the actual change was made.

Reply:
ok